src/modules/identity/RadixRemoteIdentity.ts

```diff
@@ -0,0 +1,108 @@
+import RadixECIES from '../crypto/RadixECIES'
+import RadixIdentity from './RadixIdentity'
+import RadixAccount from '../account/RadixAccount'
+import RadixSignature from '../atom/RadixSignature'
+import RadixSerializer from '../serializer/RadixSerializer'
+
+import { RadixAtom, RadixKeyPair } from '../RadixAtomModel'
+
+export default class RadixRemoteIdentity extends RadixIdentity {
+    private TOKEN: string
+    private REMOTE_URL: string
+
+    constructor(readonly keyPair: RadixKeyPair, token, host = 'localhost', port = '54345') {
```

Comment:
Do we use '54345' as a general default (ws) port in the Radix network? We use, for example, 20000 and 30000 as default port for our different networks (see universe_*.ts config files). Is this a similar "general default port"?

Reply:
This is not the Radix network, this is for connections from browsers and other applications to the localhost server on the wallet, so there's no reason this port should match any of the ports in the network. Given that, I chose one somewhat arbitrarily, it's less likely to conflict with anything than something like 20000, and it's easy to remember

Reply:
Valid point.